zerver/tests/test_custom_profile_data.py

```diff
@@ -527,6 +539,7 @@ def test_update(self) -> None:
         self.assertEqual(field.hint, "New contact number")
         self.assertEqual(field.field_type, CustomProfileField.SHORT_TEXT)
         self.assertEqual(field.display_in_profile_summary, True)
+        self.assertEqual(field.required, True)
```

Comment:
I'm in the process of merging this; adding an assert that this is `False` earlier in the test; otherwise, you've not verified that you actually changed it.

See https://zulip.readthedocs.io/en/latest/testing/philosophy.html for some other helpful advice on writing good tests.